plugin.rb

```diff
@@ -463,6 +463,8 @@ def accepted_answer_post_info
         else
           nil
         end
+
+        postInfo[3] = SiteSetting.display_name_on_posts == true ? postInfo[3] : nil
```

Comment:
Do you need the `== true` here? 
`postInfo[3] = SiteSetting.display_name_on_posts ? postInfo[3] : nil`